luigi/static/visualiser/index.html

```diff
@@ -229,6 +229,12 @@
             margin-left: 4px;
           }
 
+          #workerList .box-tools > span.label-unread-worker-messages {
+            margin-right: 6px;
+            font-style: italic;
+            vertical-align: middle;
```

Comment:
What color do you have it in? I was thinking of maybe making it red, as it's something that under normal conditions quickly disappears.